Bottom: 2798b7900084731a98e3a7d224ecbb6dd15a5655
Top:    d0c28288a5ca610503d4a2ec8c49fa456e47ca8f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-03 00:17:17 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index d387f43..0b03204 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
